torch/csrc/autograd/functions/comm.cpp

```diff
@@ -95,10 +94,8 @@ variable_list Gather::apply(variable_list&& inputs) {
   std::shared_ptr<Node> grad_fn;
   // compute this before moving variables from `inputs`
   if (compute_requires_grad(inputs)) {
-    // NOLINTNEXTLINE(cppcoreguidelines-init-variables)
```

Comment:
Is this no longer an issue? I remember this was a false positive with clang-tidy.